client/packages/common/src/intl/locales/en/common.json

```
@@ -9,6 +9,7 @@
   "auth.permission-denied": "Permission denied",
   "breadcrumb.item": "#{{id}}",
   "button.add-item": "Add Item",
+  "button.add-immunisation": "Add",
```

```suggestion
  "button.add-immunisation": "Add Immunisation",
```

Or 
```suggestion
  "button.add": "Add",
```

I think I quite like the second one, saves a bunch of translation and there's enough context from the page what we're creating I think?